fotoobo/fortinet/fortinet.py

```diff
@@ -122,7 +131,6 @@ def api(  # pylint: disable=too-many-arguments
         except requests.exceptions.ConnectionError as err:
             log.debug(err)
             error = "Unknown connection error"
-
```

Comment:
Please keep this line -> clearer layout (blank line before Python keyword leading to a block)

Reply:
Left the blank line.
The same was on line 115, where I also added an additional blank line.